packages/design-tokens/index.ts

```diff
@@ -4,5 +4,11 @@ export * from "./src/types"
 export * from "./src/ThemeManager"
 export * from "./react"
 
-// Utils are namespaced because they are potentially useful but unrelated to direct usage of design tokens.
-export * as Utils from "./src/utils"
+export {
+  mapLeafsOfObject,
+  makeCSSVariablesOfTheme,
+  flattenObjectToCSSVariables,
+  makeCSSVariableTheme,
+  objectPathToCssVarIdentifier,
+  objectPathToCssVarReference,
+} from "./src/utils"
```

Comment:
Being more explicit with these exports should make it easier to rename things in the future within utils, and potentially move stuff out of that single file.
Still thinking about which of these to export though...probably not everything here.